packages/veritone-json-schemas/schemas/vtn-standard/master.json

```diff
@@ -0,0 +1,223 @@
+        },
+        "sourceEngineName": {
+          "type": "string",
+          "description": "Engine name used to generate output (provided by Veritone) // TO DO: Verify if this is internal or external engine name",
+          "example": "capio"
+        },
+        "taskPayload": {
+          "type": "object",
+          "description": "JSON key/value task payload describing the tasks used to start the engine run (provided by Veritone) // TO DO: LINK TO DOCS EXPLAINING TASK PAYLOAD",
+          "example": {
+            "detectionThreshold": 0.7,
+            "processingFPS": 30
+          }
+        },
+        "taskId": {
+          "type": "string",
+          "description": "<GUID> of the task (provided by Veritone)",
+          "example": "6b9cd86e-de5c-4795-b605-a9b3b29fa3cf"
+        },
+        "generatedDateUTC": {
+          "type": "string",
+          "description": "DateTime (ISO8601) of when the engine result was generated (provided by Veritone if not included)",
+          "format": "date-time",
+          "example": "2017-12-08T17:19:02Z"
+        },
+        "externalSourceId": {
+          "type": "string",
+          "description": "Vendor specific referenceId to this engine result (optional)",
+          "example": "1"
+        },
+        "validationContracts": {
+          "type": "array",
+          "items": [
+            {
+              "type": "string",
+              "enum": [
+                "audio-fingerprint",
+                "ocr",
+                "face",
+                "object",
+                "transcript"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "confidence": {
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "description": "Confidence of the result expressed as a range of 0.0 to 1.0"
+    },
+    "objectCategory": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "class": {
+            "type": "string",
+            "description": "The class of the object recognized",
+            "example": "For example, if the label of an object was dog, the class of the object could be animal"
+          },
+          "@id": {
+            "type": "string",
+            "description": "Private id variable used by the engine to refer to its internal dataset",
+            "example": "1"
+          }
+        },
+        "required": [
+          "class"
+        ]
+      },
+      "description": "Optional category for object recognition"
+    },
+    "objectResult": {
+      "type": "object",
+      "properties": {
+        "boundingPoly": {
+          "$ref": "#/definitions/boundingPoly"
+        },
+        "confidence": {
+          "$ref": "#/definitions/confidence"
+        },
+        "entityId": {
+          "type": "number"
+        },
+        "entityIdentifierId": {
+          "type": "number"
+        },
+        "libraryId": {
```

Comment:
We decided that we'd like to remove this from the spec since it's duplicate information with the already-unique `entityId`.  So the best engine developers can do by writing this value is not screw up.